backend/src/user/user.controller.ts

```diff
@@ -40,7 +56,9 @@ export class UserController {
 
   @StaffOnly()
   @Post('employee/:userId/create')
-  async createEmployee(@Param('userId') userId: number): Promise<UserProfile> {
+  async createEmployee(
```

Comment:
maybe rename the same way as the hooks in the frontend.

Reply:
fixed in latest commit